subsys/bluetooth/common/Kconfig

```
@@ -183,12 +191,44 @@ config BT_BUF_CMD_TX_SIZE
 	  to the maximum of 255.
 
 config BT_BUF_CMD_TX_COUNT
-	int "Number of HCI command buffers"
-	default 2
-	range 2 64
+	int "Number of HCI command buffers" if !BT_HCI_RAW || !HAS_BT_CTLR
+	# !BT_HCI_ACL_FLOW_CONTROL ==> Permit configurable Num_HCI_Command_Packets for
+	# Host-only build but hidden when HAS_BT_CTLR=y (default 1 for Controller-only build).
+	# !(BT_HCI_RAW && HAS_BT_CTLR) ==> Host-only build.
+	depends on !BT_HCI_RAW || !HAS_BT_CTLR || !BT_HCI_ACL_FLOW_CONTROL
```

This is a bit difficult to read. Maybe something like this?
```suggestion
	# This option is only visible for a user when Host and Controller are build together, or for
	# Host-only builds.
	int "Number of HCI command buffers" if !BT_HCI_RAW || !HAS_BT_CTLR
	# This option is present when Host and Controller are build together, or
	# for Host only builds, or when Controller-to-Host ACL data flow control
	# is disabled.
	depends on !BT_HCI_RAW || !HAS_BT_CTLR || !BT_HCI_ACL_FLOW_CONTROL
```